src/common/HostInterface.c

```diff
@@ -528,7 +525,16 @@ void ProcessCommandFromHost(char * strCMD)
 
 	if (strcmp(strCMD, "DEBUGLOG") == 0)
 	{
-		DoTrueFalseCmd(strCMD, ptrParams, &DebugLog);
+		BOOL enable_files = ardop_log_is_enabled_files();
+		if (DoTrueFalseCmd(strCMD, ptrParams, &enable_files)) {
+			ardop_log_enable_files((bool)enable_files);
+			snprintf(cmdReply, sizeof(cmdReply), "%s now %d", strCMD, enable_files);
+			ZF_LOGI("DEBUGLOG now %d", enable_files);
+			SendReplyToHost(cmdReply);
+		}
+		else {
+			snprintf(strFault, sizeof(strFault), "Syntax Err: %s %s", strCMD, ptrParams);
+		}
```

Comment:
DoTrueFalseCmd already did SendReplyToHost() or set strFault.  As written, this sends duplicates, and this SendReplyToHost produces "DEBUGLOG now 1" or "DEBUGLOG now 0" whereas DoTrueFalseCmd() produces the better "DEBUGLOG now TRUE" or "DEBUGLOG now FALSE".

See also the comment on return value from DoTrueFalseCmd.  As written, it sends "FAULT Syntax Err: DEBUGLOG (null)" as well as the correct response of "DEBUGLOG TRUE" or "DEBUGLOG FALSE" when no parameter is provided with a DEBUGLOG command.

Reply:
Sure enough. I have put the DEBUGLOG logic back the way it was before.

I have found an unrelated error-reporting bug in `DoTrueFalseCmd()` as it exists on develop. Due to conflicts, I'm going to wait until after this PR is merged to submit a fix for it.